Hosts/VS/TddStud10App/packages.config

```diff
@@ -3,6 +3,7 @@
   <package id="BuildTools.FxCop" version="1.0.1" targetFramework="net45" />
   <package id="CommonServiceLocator" version="1.3" targetFramework="net45" />
   <package id="EditorUtils2010" version="1.5.0.1" targetFramework="net45" />
+  <package id="FSharp.Core" version="3.1.2.5" targetFramework="net45" />

```

Comment:
It is a little confusing, but the NuGet package refers to the F# version

I.e. v3.1.2.5 of the NuGet package is for F# 3.1, i.e. version 4.3.1.0 of FSharp.Core 

Version 4.0.0.1 of the NuGet package is for F# 4.0, i.e. version 4.4.0.0 of FSharp.Core